http.c

```diff
@@ -2476,7 +2525,9 @@ evhttp_connection_base_bufferevent_new(struct event_base *base, struct evdns_bas
 {
 	struct evhttp_connection *evcon = NULL;
 
-	event_debug(("Attempting connection to %s:%d\n", address, port));
+	event_debug(("%s %s:%d\n",
+	             (dnsbase?"New connection from":"Attempting connection to"),
+	             address, port));
```

Comment:
My initial commit was : https://github.com/miniupnp/libevent/commit/8de03d1a18aa8f9068f75da950ce99368f67e267
It is to have different message when being the HTTP server or the HTTP client.

Reply:
I think at some point it was squashed in @errzey commit... I don't remember why/how, that's 3 years ago :(

Reply:
Yeah, the birth of this patch is old enough, but thanks for getting back to it!
So this hunk should be dropped

Reply:
I don't see how `dnsbase == "connection from"`, I can call `evhttp_connection_base_bufferevent_new(NULL, evdns_base_new(), ...)`

Reply:
When I wrote this patch 3 years ago it fixed the bug.
I'm open for a better way to detect whether the http connection is a server or a client one

Reply:
I guess the `event_debug()` line was written when only HTTP client was implemented. HTTP server was later implemented but the debug message was not updated.

By the way, you're right it looks like the test doesn't work anymore.
I think it would be better to remove the log from the `evhttp_connection_base_bufferevent_new()` function and set it in the calling functions.

Reply:
>When I wrote this patch 3 years ago it fixed the bug.

This hunk cannot fix any bug.
So I think that this should be removed.

>I'm open for a better way to detect whether the http connection is a server or a client one

Do you really use debug logging in libevent? I guess not.
And even if you do, this is a separate issue, totally unrelated to this PR

Reply:
yes it is unrelated to this PR.
But yes I used debug logging in libevent while I was adding custom HTTP request support.

Reply:
>I think it would be better to remove the log from the evhttp_connection_base_bufferevent_new() function and set it in the calling functions.

First of all, like I wrote in the previous comment, let's keep this PR clean to the unrelated changes.

But to make it clear, evhttp_connection_base_bufferevent_new() is an exported function, so if you will move the logging message into the calling function it will be lost for the users that calls it directly (and it is pretty common).